src/rpc/mission.rs

```diff
@@ -165,6 +166,35 @@ impl MissionService for MissionRpc {
         let res = self.request("removeGroupCommandItem", request).await?;
         Ok(Response::new(res))
     }
+
+    async fn get_session_id(
+        &self,
+        _request: Request<mission::v0::GetSessionIdRequest>,
+    ) -> Result<Response<mission::v0::GetSessionIdResponse>, Status> {
+        let response = self
+            .get_user_flag(Request::new(trigger::v0::GetUserFlagRequest {
+                flag: String::from("dcs_grpc_session_id"),
+            }))
+            .await?
+            .into_inner();
+        let current_session_id = response.value as i64;
+        if current_session_id == 0 {
+            let ts = OffsetDateTime::now_utc().unix_timestamp();
+            let _result = self
```

Comment:
I think the following can be removed
```suggestion
            self
```

Reply:
Good point, updated